dashboards/grafana/README.md

```diff
@@ -0,0 +1,197 @@
+* [MP Metrics](https://helidon.io/docs/latest/mp/metrics/metrics)
+* [MP REST Metrics](https://helidon.io/docs/latest/mp/guides/metrics#controlling-rest-request-metrics)
+* [Helidon Labs Virtual Threads - JDK24](https://github.com/helidon-io/helidon-labs/tree/main/incubator/virtual-threads-metrics)
+
+## Helidon Setup
+
+### Application Identification
+
+To ensure we can identify individual helidon applications as well as allowing drill-through, you should add the
+following labels to your generated metrics using the system property or `application.yaml` (SE) or `microprofile-config.properties` (MP):
+
+**Helidon SE**
+
+* System Property: `-Dmetrics.app-name=my-se-app"`
+* `application.yaml`: 
+  ```bash
+  metrics:
+    app-name: "my-se-app"
+  ```
+
+This will add the label to the output for each metric, e.g.:
+
+```bash
+jvm_uptime_seconds{instance="host.docker.internal:7001", job="helidon", scope="base", app="my-se-app"}
+```
+
+**Helidon MP**
+
+* System Property: `-Dmp.metrics.appName=my-mp-app`
+* `microprofile-config.properties` `
+   ```bash
+   mp.metrics.appName=my-mp-app
+   ```
+
+This will add the label to the output for each metric, e.g.:
+
+```bash
+REST_request_seconds{... mp_app="my-mp-app", mp_scope="base", quantile="0.5"}
+```
+
+### Required Dependencies
+
+**SE**
+
+```xml
+<dependency>
+    <groupId>io.helidon.webclient</groupId>
+    <artifactId>helidon-webclient-metrics</artifactId>
+</dependency>
+
+<dependency>
+    <groupId>io.helidon.webserver.observe</groupId>
+    <artifactId>helidon-webserver-observe-metrics</artifactId>
+</dependency>
+
+<dependency>
+    <groupId>io.helidon.metrics</groupId>
+    <artifactId>helidon-metrics-system-meters</artifactId>
+    <scope>runtime</scope>
+</dependency>
+```
+
+**MP**
+
+Include the following:
+```xml
+<dependency>
+    <groupId>io.helidon.microprofile.metrics</groupId>
+    <artifactId>helidon-microprofile-metrics</artifactId>
+</dependency>
+
+<dependency>
+    <groupId>io.helidon.webclient</groupId>
+    <artifactId>helidon-webclient-metrics</artifactId>
+    <scope>runtime</scope>
+</dependency>
+
+<dependency>
+    <groupId>io.helidon.metrics</groupId>
+    <artifactId>helidon-metrics-system-meters</artifactId>
+    <scope>runtime</scope>
+</dependency>
+```
+
+To include additional, experimental virtual threads metrics, available only in JDK 24, you can clone Helidon Labs at
+https://github.com/helidon-io/helidon-labs.git and build and include the following:
+
+```xml
+<dependency>
+    <groupId>io.helidon.labs.incubator</groupId>
+    <artifactId>helidon-labs-incubator-virtual-threads-metrics</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <scope>runtime</scope>
+</dependency>
+```
+
+### Enabling Metrics
+
+Additional properties are required to ensure we get the full range of metrics:
+
+> Note: These are shown as system properties but can be set using the appropriate way in SE or MP.
```

Comment:
"These are shown as config file settings but can be set in any relevant SE or MP config source."